src/engine/runtime.js

```diff
@@ -2560,6 +2560,17 @@ class Runtime extends EventEmitter {
     addDataFileRow(fileName) {
```

Comment:
are these methods still needed since we moved to the in extension function calls